### DISCUSSION TITLE
Best way to sync releases and flux-framework webpage?

### DISCUSSION THREAD

**Discussion by dongahn** [flux-framework/flux-core#4191] [General]
I see the latest releases listed at https://flux-framework.org are pretty old. What will be the best way to synchronize our releases and webpage update? We could make the webpage update as a requirement in https://github.com/flux-framework/flux-core/wiki/Release-Checklist. But there could be a more automated way to do this though.

**Comment by grondo** [comment]
We should be able to do this via a github workflow that runs after Releases are created. It is just a matter of someone taking the time to implement it.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release
